app/views/posts/_post.html.erb

```diff
@@ -0,0 +1,23 @@
+<ul class="posts">
+  <li class="post">
+    <%= link_to user_post_path(post.author_id, post.id) do %>
+      <div class="post_container">
+        <div class="post_title"><%= post.title %></div>
+        <div class="post_text"><%= post.text %></div>
+        <div class="post_indicators">
+          <div class="post_comments_counter">Comments: <%= post.comments_counter %></div>
+          <div class="post_likes_counter">Likes: <%= post.likes_counter %></div>
+        </div>
+      </div>
+    <% end %>
+    <ul class="post_comments">
+    <% if post.comments.empty? %>
+      <li>No comments</li>
+    <% else %>
+      <% post.recent_5_comments.each do |comment| %>
+        <%= render comment %>
```

Comment:
* Kindly make sure your app handles all routes as shown in the wireframe given, as after adding users using the rails console i still can't create a post and cant view either. ensure you are able to test your views at your end 
![image](https://user-images.githubusercontent.com/52497246/193430082-44270672-063d-439a-a449-b16320d8ed5a.png)
![image](https://user-images.githubusercontent.com/52497246/193430111-a3c10271-26d6-4e88-a00f-983f680171ff.png)

Reply:
Hello @Davidosky007, Thanks for the review but I am sorry, I am a little bit confused.   I clearly see the issues you are mentioning but on my side, everything is working as it should. I can create posts even when all routes are implemented and all working well. could you help me to investigate further more why this is happening? That way I can be awere of which part to change

Reply:
Are you free for a call?

Reply:
Yes, I am

Reply:
As you can see from the screenshot I shared I couldn't create a post

Reply:
Yes, I am

Reply:
my zoom user name => " Brenda Wihogora"

Reply:
My slack handle is @davidosky

Reply:
@future CR all works fine now 😃 ensure you populate the database with data 👍